[Bug]: build error after running iframe-to-component schematic

### Package name

third-party

### Package version

11.0.0

### Reproduction steps

```
npm create @o3r test-3rd-party
cd test-3rd-party/
npm run ng g app webapp
npm run ng g component comp
npm run ng add @o3r/third-party --project-name=webapp
npm run ng g iframe-to-component -- --path=apps/webapp/src/components/comp/comp.component.ts
npm run build -w webapp
```

### Current result

```
> webapp@0.0.0-placeholder build
> ng build

Application bundle generation failed. [3.525 seconds]

✘ [ERROR] TS2564: Property 'bridge' has no initializer and is not definitely assigned in the constructor. [plugin angular-compiler]

    apps/webapp/src/components/gigi/gigi.component.ts:18:12:
      18 │     private bridge: IframeBridge;
         ╵             ~~~~~~


✘ [ERROR] TS2304: Cannot find name 'ViewChild'. [plugin angular-compiler]

    apps/webapp/src/components/gigi/gigi.component.ts:19:5:
      19 │     @ViewChild('frame')
         ╵      ~~~~~~~~~


✘ [ERROR] TS2552: Cannot find name 'ElementRef'. Did you mean 'Element'? [plugin angular-compiler]

    apps/webapp/src/components/gigi/gigi.component.ts:20:19:
      20 │     private frame: ElementRef<HTMLIFrameElement>;
         ╵                    ~~~~~~~~~~

  'Element' is declared here.

    node_modules/typescript/lib/lib.dom.d.ts:7894:12:
      7894 │ declare var Element: {
           ╵             ~~~~~~~


```

### Expected result

 `Application bundle generation complete. [2.156 seconds]`

### Additional comments

There are some issues in the schematic templates.

--- a/packages/@ama-sdk/schematics/package.json
+++ b/packages/@ama-sdk/schematics/package.json
@@ -128,7 +128,7 @@
   },
   "generatorDependencies": {
     "@swc/cli": "~0.4.0",
-    "@swc/core": "~1.6.0",
+    "@swc/core": "~1.7.0",
     "@swc/helpers": "~0.5.0",
     "@commitlint/cli": "^19.0.0",
     "@commitlint/config-conventional": "^19.0.0",

